README.md

```diff
@@ -19,60 +11,4 @@ An open source interface for Uniswap -- a protocol for decentralized exchange of
   - [V1](https://hackmd.io/C-DvwDSfSxuh-Gd4WKE_ig)
   - [V2](https://uniswap.org/whitepaper.pdf)
   - [V3](https://uniswap.org/whitepaper-v3.pdf)
```

Comment:
Can we inline these into this repository and remove the external links?  Or perhaps pin them on IPFS?